src/components/Dialog.tsx

```diff
@@ -186,13 +186,19 @@ const slideOutRight = keyframes`
 
 const fadeIn = keyframes`
```

Comment:
These are now generic enough that I think `fadeIn`/`fadeOut`/`fadeAnimationCss` can all be moved to `theme/animations.ts`.